src/passes/fixFunctionCallReferences/index.ts

```diff
@@ -0,0 +1,18 @@
+import { AST } from '../../ast/ast';
+import { ASTMapper } from '../../ast/mapper';
+
+import { FunctionDefinition } from 'solc-typed-ast';
+
+import { GetGetters } from './getGetters';
+import { FixFnCallRef } from './fixFnCallRef';
+
+export class FixFunctionCallReferences extends ASTMapper {
```

Comment:
Instead of having a separate top-level pass for this, I think it would be better to combine GettersPublicStateVars and FixFnCallRef into a compound pass like this one, with GettersPublicStateVars building up the getterFunctions map as it goes. This would also require the compound pass to run each pass using their map function rather than applying each one to each root